### DISCUSSION TITLE
Can I disable this pop-up?

### DISCUSSION THREAD

**Discussion by josepheiba** [PlayCover/PlayCover#648] [Q&A] [answered]
Every time I switch back from another app to a game it shows this pop-up, is there anyway to disable it?
<img width="555" alt="Screenshot 2022-12-28 at 3 24 37 PM" src="https://user-images.githubusercontent.com/101673502/209827173-d93a7fae-9880-4a07-96f9-55b881f7d2f2.png">

**Comment by amirsaam** [comment]
There is no way to disable it. Game tries to connect to Game Centre that is blocked in sideloaded games

**Comment by josepheiba** [comment, marked as answer]
I have just discovered that if I double click the game then immediately switch to another app (making the game open in the background) I don't get the initial pop-up nor does it keep bothering me every time I switch back to the game nor does it takes 5 seconds to give me control of the game.
